src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -230,13 +230,14 @@ public static string SelectProductNameForDirectory(Platform.XDG_Type dirpath)
         {
             //TODO: XDG_DATA_DIRS implementation as per GitHub issue #1060
 
+            string defaultHomePath = System.Environment.GetEnvironmentVariable("HOME") ?? Directory.GetCurrentDirectory();
```

Comment:
Somebody met this before:http://unix.stackexchange.com/questions/207469/systemd-permission-issue-with-mkdir-execstartpre

From the answer, 2 options;
1. Add "PermissionsStartOnly=true" to service
2. Use RuntimeDirectory and RuntimeDirectoryMode